### ISSUE TITLE
Add the radical U+221A and the pi under u+‎1D6E1 for Calculator app

### ISSUE THREAD

**Issue by MariannaPaszkowska** [googlefonts/googlesans-flex#1020]
Adding √ and π encoded as 𝛡, for now. 
This is because of the subsetting concerns.

**Comment by ryankiley** [comment]
These are looking great in the test file @chrissimpkins gave me. The pi symbol in found in the glyphs panel of Illustrator is `𝛡`, not `π` right? 

Here's a WIP of potentially what they might look like in Calculator. [image redacted]

**Comment by chrissimpkins** [comment]
> The pi symbol in found in the glyphs panel of Illustrator is 𝛡, not π right?

That's correct. We used ` ‎1D6E1 MATHEMATICAL BOLD PI SYMBOL` to encode the pi symbol that we used for the Calculator UI so that it does not cover the Greek language encoding that will be used for Greek language text.  This will be undesirable GS Flex to Google Sans fallback in Greek texts.  We can consider changing to the default pi symbol when GS Flex includes full Greek support and we no longer need the GS fallback.
